File: C++/55.cpp
// Remove All characters except alphabets
#include <bits/stdc++.h>
using namespace std;

int main() {
	string s;
	cin>>s;
	string ans="";
	for(auto c:s){
		if((c>='A'&&c<='Z')||(c>='a'&&c<='z')){
			ans+=c;
		}
	}
	cout<<ans;
	return 0;
}

File: Python/55.py
# 55. Program to Remove all Characters in a String Except Alphabets

String = input("Enter a string: ")

# >>>Enter a string: Program to Remove all characters in a String @@@@$^*(@^(*^%# except Al%(*&phab*(@$ets
#	 ProgramtoRemoveallcharactersinaStringexceptAlphabets 

alphabets_up = [chr(alpha) for alpha in range(65, 91)]
alphabets_low = [chr(alpha) for alpha in range(97, 123)]

ans_string = '' #String can't be mutated

for character in String:
    if character in alphabets_low or character in alphabets_up:
        ans_string += character

print(ans_string)